pkg/TIPA.py

```
@@ -492,14 +505,14 @@ def process_image(self, attempt: int, temp_files: tuple, is_inventory: bool) ->
             self.show_image(image, "final_image", "Showing final image")
 
         return image
-    
+
     def validate_wordlist(self, wordlist: list) -> bool:
         if len(wordlist) == 0 or (len(wordlist) == 1 and len(wordlist[0]) <= 2):
             return False
         if wordlist[0] == "Body":
             return False
         return True
```

Ruff (0.8.0)

Return the condition `not wordlist[0] == "Body"` directly

Replace with `return not wordlist[0] == "Body"`

(SIM103)